### PR TITLE
request for pull pls pls pls pls mama

### PR DESCRIPTION
**Pull request by varadfromeast**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,4 +22,4 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
-This is added by Manoj Kulkarni
```
Comment (manojkulkarni) [inline README.md]: Please do not remove authors name

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "testHeroku",
+  "name": "Hercules",
```
Comment (manojkulkarni) [inline package.json]: This looks fine to me

File: README.md (hunk 1 above)
Comment (varadfromeast) [reply README.md]: no i want to steal your work

File: package.json (hunk 2 above)
Comment (varadfromeast) [reply package.json]: ok

**Pull request merged**